org.eclipse.editorconfig.ui/src/main/java/org/eclipse/editorconfig/ui/internal/EditorConfigVisitor.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Nathan Jones, Jackson Bailey
```

Comment:
The tests have never been run on Windows. I have tried to fix the platform-dependent nature of the test code in the "cross-platform-functional-tests" branch by using `Paths.get(URI)` instead of `Paths.get(String[])`. Can you please pull that branch and see if it works for you now?